### DISCUSSION TITLE
Версия 13.1 настройка

### DISCUSSION THREAD

**Discussion by dumbodumbos** [hufrea/byedpi#106] [Q&A]
Добрый. Вопрос по настройке.  
Версия 12 - пускалась вот с такой настройкой .ciadpi-x86_64 --disorder 1 --fake 0 --ttl 1 -A torst --tlsrec 1+s --debug 1 & все работало как и с MACOS так и с IOS. 
Обновил до версии 13.1 - на MACOS перестало работать - Safari не может отобразить станицу. На устройствах IOS - все работает штатно ( открывается как и youtube так и обычные страницы). Попробовал следующий конфиг .ciadpi-x86_64 --disorder 1 --fake -1 --md5sig --ttl 1 -A torst --tlsrec 1+s --debug 1 &. Эффект тот же. 

Подскажите - с каким ещё конфигом можно попробовать ?

**Comment by jura1243** [comment]
я запускаю на линуксе так:  ./ciadpi --proto http --auto=none --disorder 1  --auto=torst --tlsrec 1+s &
не работает касперский и андроид тв.

**Reply by smokingplaya** [reply]
привет

**Comment by dumbodumbos** [comment]
Пропробую. Ни касперского и андройт ТВ не использую. Но надо что бы работал сафари

**Comment by Skullckoor** [comment]
`**так и с IOS**`
Простите, а как на iOS поставить?

**Comment by xXDeMoSiKXx** [comment]
На устройствах IOS - все работает штатно.


Приветствую! Можете пожалуйста подсказать как на IOS можно использовать?
jailbreak / trollstore присутствует. 
Максимально был бы благодарен, я вменяемый, проблем с осознанием не будет)

**Reply by zimbabwe303** [reply]
Прямо на iPhone никак: никто так за это время ничего и не сделал для этого. Можно запустить на любом другом устройстве в локальной сети, а в браузере на iPhone указать его IP и порт byedpi (по умолчанию 1080). Ещё чисто теоретически можно было бы откомпилировать byedpi для ARM, но там же ядро не Linux, как в Android, так что нужно делать полноценный порт.

**Reply by xXDeMoSiKXx** [reply]
Я вас понял, спасибо за ответ! Да по локалке не интересует к сожалению, дома все прекрасно работает на пк/телевизоре и на iPhone.
А вот на улице/области очень требуется, в последнее время плохой интернет сам по себе, а там ещё плюсом впн, ничего не посмотришь нормально ( YouTube ), и нормально не поговоришь ( Discord ). 
Ладно, надеюсь что нибудь когда нибудь придумают🥲

**Reply by zimbabwe303** [reply]
Есть Unicorn HTTPS: https://apps.apple.com/us/app/unicorn-https/id1466584968

**Reply by bananakid** [reply]
[Rumble (ByeDPI) для iOS 9.0+ 64-bit](https://iklassika.ru/app.php?id=5833)

**Comment by zimbabwe303** [comment]
Мой нынешний вариант настроек, работающий и для FreeTube:
`-s1 -q1:1:3 -Y -At -T2 -b1000 -S -f-1 -r1+s:0:1 -As`
(обновил -r на менее проблемный вариант)

В отличие от предыдущего (`-s1 -q25 -Y -At -T2 -b1000 -S -f-1 -r1+sm -As`), этот вообще не тормозит при старте видео, и вообще не выдаёт никаких ошибок в командной строке. Думаю, вместо 3 в `-q1:1:3`, можно поставить и большее число (это число пропущенных пакетов).

**Reply by evgeniy153** [reply]
Bayou

**Reply by zimbabwe303** [reply]
Немного обновлённый стабильный вариант настроек (работает с апреля до сегодняшнего дня):
`-s2:1 -q2:1:5 -Y -At -T2 -b1000 -S -f-1 -r1+s -As`
